docs/pages/database-access/architecture.mdx

```diff
@@ -55,11 +55,11 @@ connect to a database.
    a client certificate. See [Issuing User Certificates](../architecture/authentication.mdx)
    for more details on how it works.
 2. The user picks the database they want to connect to from the list of available
-   databases shown in `tsh db ls` command and retrieves a short-lived X.509
-   certificate for it with `tsh db login`.
-3. The user uses a standard database client (e.g. `psql`, `mysql` or one of the
-   [graphical clients](./guides/gui-clients.mdx)) to connect to the Proxy, authenticating
-   with the client certificate from step 2.
+   databases shown in `tsh db ls` command.
+3. The user connects to the database with the `tsh db connect` command, which
+   first retrieves a short-lived X.509 certificate and then launches the
+   standard database client (e.g. `psql`) with this client certificate to
+   authenticate with the Proxy.
```

Comment:
```suggestion
   authenticate with the Teleport Proxy Service.
```
To be a bit more precise, which architecture is often a place where new users get confused